addons/binding/org.openhab.binding.minecraft/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,205 @@
+            <channel id="playerLocationX" typeId="playerLocationX"/>
+            <channel id="playerLocationY" typeId="playerLocationY"/>
+            <channel id="playerLocationZ" typeId="playerLocationZ"/>
+        </channels>
+
+        <config-description>
+           <parameter name="playerName" type="text" required="true">
+               <label>Player name</label>
+               <description>The name of player</description>
+           </parameter>
+       </config-description>
+    </thing-type>
+    
+    <thing-type id="redstoneSign">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="server"/>
+        </supported-bridge-type-refs>
+    
+        <label>Sign</label>
+        <description>Sign with redstone going into block under it</description>
+
+        <channels>
+            <channel id="signActive" typeId="signActive"/>
+        </channels>
+
+        <config-description>
+           <parameter name="signName" type="text" required="true">
+               <label>Sign name</label>
+               <description>The text on sign</description>
+           </parameter>
+       </config-description>
+    </thing-type>
+    
+    <channel-type id="players">
+        <item-type>Number</item-type>
+        <label>Players</label>
+        <description>The number of players on server.</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="maxPlayers">
+        <item-type>Number</item-type>
+        <label>Max Players</label>
+        <description>The maxumum number of players on server.</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="bukkitVersion">
+        <item-type>String</item-type>
+        <label>Bukkit Version</label>
+        <description>The bukkit version running on server.</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="version">
+        <item-type>String</item-type>
+        <label>Version</label>
+        <description>The Minecraft version running on server.</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="name">
+        <item-type>String</item-type>
+        <label>Name</label>
+        <description>Name of minecraft server</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="online">
+        <item-type>Switch</item-type>
+        <label>Online</label>
+        <description>Servers online status.</description>
+        <category>Switch</category>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="playerName">
+        <item-type>String</item-type>
+        <label>Name</label>
+        <description>Name of minecraft player</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="playerOnline">
```

Comment:
I'd think this could be removed and the "online" channel-type could be used for the player as well.